data/remnant/remnant 2 cognizance.txt

```diff
@@ -1357,7 +1357,7 @@ mission "Remnant: Cognizance 31"
 
 mission "Remnant: Cognizance 32"
 	name "Pick Up Refined Fuel"
-	description "Chilia has asked you to pick up 58 tons of refined fuel from <planet> by <date>."
+	description "Chilia has asked you to pick up 48 tons of refined fuel from <planet> by <date>."
```

Comment:
It is required as otherwise it won't fit on the peregrine as it was before the peregrine was buffed without justification (and without mentioning this buff anywhere on the PR).

Reply:
Side note, why does this mission use a substitution for <planet>, but not for the amount of cargo?

```suggestion
	description "Chilia has asked you to pick up <tons> of refined fuel from <planet> by <date>."
```